packages/block-editor/src/components/block-mover/mover-description.js

```diff
@@ -231,43 +234,88 @@ export function getMultiBlockMoverDescription(
 	firstIndex,
 	isFirst,
 	isLast,
-	dir
+	dir,
+	orientation
 ) {
 	const position = firstIndex + 1;
 
-	if ( dir < 0 && isFirst ) {
-		return __( 'Blocks cannot be moved up as they are already at the top' );
-	}
+	if ( dir < 0 && ! isFirst ) {
+		// moving up
+		const movementDirection = getMovementDirection( 'up', orientation );
```

Comment:
I think this function can also return 'right' if `isRTL()` is `true`, but there isn't an `if` statement below for 'right'.

It's probably easiest to replicate the logic from the `getBlockMoverDescription`. It could use the same ordering of `if` statements.

I also wonder whether `getMultiBlockMoverDescription` should have a description for when all blocks are selected and can't be moved in any direction. A bit like how `getBlockMoverDescription` has a case for when the only block is selected.

Reply:
Thanks for pointing out the missing condition! I updated `getMultiBlockMoverDescription` with reference to `getBlockMoverDescription`.

Reply:
That's great, thanks!